src/asm/rgbasm.5

```diff
@@ -647,7 +683,9 @@ AOLer    EQUS  "Me too"
 .Pp
 Note that string symbols that are part of a
 .Ic PURGE
-command WILL NOT BE EXPANDED as the ONLY exception to this rule.
+command
+.Em WILL NOT BE EXPANDED
+as the ONLY exception to the rule.
```

Comment:
As a point of style I would lowercase text if I changed it to italics.

Reply:
Will do.